### ISSUE TITLE
As Hypothesis I want to be able to capture users comms opt-in consent on signup.

### ISSUE THREAD

**Issue by dawachan** [hypothesis/product-backlog#431]
_Justification:_ We are unable at present to email our users for marketing, product or user research purposes and it is proposed that we introduce a form field on sign up to capture this opt-in preference which will unblock us from being able to contact our user base for research/testing, marketing and product improvements.

**High level SOW**

Add a checkbox with to the login screen to: https://hypothes.is/signup

Acceptance criteria:

**Done when:**

- [ ] The revised sign up form contains an explicit opt-in to product and research communications checkbox with the copy "I agree to receive occasional communication for product and research purposes."- **checked by default**.

**Out of Scope:**

- Don't need to supply a way to opt-out (as it is in the marketing comms).
- Opt-in to privacy policy checkbox is covered in #611 

**Mock up:**
_Revised Sign up form with opt-in to comms consent_
![sign up - revised language - final ammended](https://user-images.githubusercontent.com/5212631/39360101-bfed902e-4a14-11e8-867f-1156cf3b82d1.png)

_Current Sign up form_
![current sign up form](https://user-images.githubusercontent.com/5212631/38312450-b4e9a03a-3819-11e8-8ea2-237937c09ea4.png)

**Comment by dawachan** [comment]
Updated the opt-in copy to say: "I agree to receive occasional communication for product and research purposes." as per @dwhly's recommendation.

**Comment by xolotl** [comment]
I'm not sure that we don't need a way for users to opt out in H. itself. I think the best solution would be for users to have communication toggles in their H. account view. All such toggles would communicate via API with whatever email provider H. is using (currently MailChimp) to create/toggle the subscription for that user's email address.

**Comment by klemay** [comment]
See also: https://github.com/hypothesis/product-backlog/issues/354

**Comment by ajpeddakotla** [comment]
| Business Viability | User Engagement | Building Community | Level of Effort |
|---|---|---|---|
| 1 | 3 | 3 | 2 |

**Comment by klemay** [comment]
Closing in favor of https://github.com/hypothesis/product-backlog/issues/1086
